### ISSUE TITLE
Unix portability

### ISSUE THREAD

**Issue by syaskovets** [mmolero/SimNDT#3]
Hi. I have been looking for a piece of software to simulate ultrasonic waves for my NDT project, and your project looks very interesting to me. The only problem is I am currently limited to unix [ubuntu] platform, and after installing all the prerequisites I encountered an error when importing the engine (SimNDT.engine.efit2dcython) which is seemingly is a python library for Win. 
In REAME it is stated that you are open to collaboration, so creating an issue on Github seemed to be the easiest way reaching you.
Have you tested the project on other platforms than Windows? And if yes, could you please give me a hint how to get it running on Unix? 
Best regards,
Serhii Yaskovets

**Comment by mmolero** [comment]
Actually this compiled function is only required if you want to launch the simulator in serial mode. I encourage to have a GPU in your machine in order to run in parallel

**Comment by wangwentao8797** [comment]
I can not add ellipse or rectangle after I create one scenario.
Could you please tell me how to solve the problem?

**Comment by mmolero** [comment]
Thanks @syaskovets

Could you please share your python environment used on Linux in order to know which library versions you are using in your Linux OS. Thanks

**Comment by wangwentao8797** [comment]
Thank you for sharing this software. Unfortunately, I can not run this software. I was wondering if there is any problem when you compile the execute the file? Do you have the old version that still works? Thanks.

**Comment by mmolero** [comment]
Could you please detail your OS environment?Have you tried to run from source?

**Comment by wangwentao8797** [comment]
Yes, I tried. My OS environment is Window 10.

**Comment by mmolero** [comment]
Thanks @wangwentao8797  I've checked that this issues does exist. I will generate a new version solving this issue.

**Comment by mmolero** [comment]
@wangwentao8797  please download the new version v0.51. If you find another issue or the same issue happens again please create a new tickect describing the problem in order to track it correctly.  I will close this one because this deals about unix portability and it seems that it was solved. 

Thanks

**Comment by syaskovets** [comment]
HI @mmolero. Sorry for a bit late response. Here is the list of libraries and their versions currently being used in my python environment:

> appdirs==1.4.3
certifi==2018.10.15
cycler==0.10.0
decorator==4.3.0
kiwisolver==1.0.1
Mako==1.0.7
MarkupSafe==1.0
matplotlib==3.0.0
mkl-fft==1.0.6
mkl-random==1.0.1
numpy==1.15.3
olefile==0.46
Pillow==5.3.0
pybind11==2.2.4
pyopencl==2018.2.1
pyparsing==2.2.1
python-dateutil==2.7.3
pytools==2018.5.2
pytz==2018.5
PyWavelets==1.0.0
scipy==1.1.0
sip==4.19.8
six==1.11.0
tornado==5.1.1


Python version is 3.6.5. The system is Ubuntu 18.04, just in case it may help. Additionally, when I fully test the software, I will make a pull request with all the changes made to run it properly under such an environment.

**Comment by wangwentao8797** [comment]
Thanks, @mmolero. The new version (V 0.51) fixed the issue unable of adding ellipse or rectangle. There is another issue remain: unable to add the boundaries. I can see the boundaries in the left column, but it does not show the boundaries in the GUI. When I try to run the simulation set up, there is an error "Please define the Boundaries Conditions!!!!"
![image](https://user-images.githubusercontent.com/12880175/48427471-c68a4200-e736-11e8-879d-f1718291ef9b.png)
Could you fix that issue too? Windows 10.
Thanks a million!

**Comment by mmolero** [comment]
@wangwentao8797 Please download the new version v0.52 that solves the boundary condition issue
For new issues  please open a new issue and not use this one. Thanks!

**Comment by wangwentao8797** [comment]
Thanks, it works!
